rclc_parameter/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ find_package(rcutils REQUIRED)
 find_package(builtin_interfaces REQUIRED)
 find_package(rcl_interfaces REQUIRED)
 find_package(rosidl_runtime_c REQUIRED)
+find_package(Threads REQUIRED)
```

Comment:
```suggestion
```

Reply:
Same here about micro-ROS if we want to build `rclc_parameter` in embedded platforms

Reply:
resolved in https://github.com/ros2/rclc/pull/237/commits/da1cebeea58447ff216323994f11e649bdd1e3a4